utils/main.go

```diff
@@ -88,6 +89,19 @@ func CheckExpired(expireOn string) (bool, error) {
 
 }
 
+// ValidateUserKey checks for the presence of a characters which should not be
+// allowed in a username string, returns an error if any such characters are
+// detected, nil otherwise.
+// Currently, the only banned characters are whitespace characters.
+func ValidateUserKey(user string) error {
+	whiteSpaceRegexp, _ := regexp.Compile("\\s")
+
+	if hasWhiteSpace := whiteSpaceRegexp.MatchString(user); hasWhiteSpace == true {
```

Comment:
Are you using `hasWhiteSpace` elsewhere?

Why not simply doing the following(I guess this is what @hopkiw meant):

```go
if whiteSpaceRegexp.MatchString(user) {
```

Reply:
fixed